SPECS/python-pip/CVE-2025-50181.patch

```diff
@@ -0,0 +1,61 @@
+From d75419bb31f1e5527773d1e22586881241c2f774 Mon Sep 17 00:00:00 2001
+From: jykanase <v-jykanase@microsoft.com>
+Date: Mon, 23 Jun 2025 13:00:58 +0000
+Subject: [PATCH] CVE-2025-50181
+
+Upstream Patch Reference: https://github.com/urllib3/urllib3/commit/f05b1329126d5be6de501f9d1e3e36738bc08857
```

Comment:
reviewing the upstream patch diff w.r.t the changes made here. Will update after a local checkout and through understanding.